bus-mapping/src/circuit_input_builder.rs

```diff
@@ -739,12 +790,17 @@ impl<'a> CircuitInputStateRef<'a> {
         let salt = step.stack.nth_last(3)?;
         let init_code = get_create_init_code(step)?;
         Ok(get_create2_address(
-            self.call().address,
+            self.call()?.address,
             salt.to_be_bytes().to_vec(),
             init_code.to_vec(),
         ))
     }
 
+    /// Check if address is a precompiled or not.
+    pub fn is_precompiled(&self, address: &Address) -> bool {
+        (1..=9).contains(address.as_bytes().last().unwrap())
```

Comment:
Isn't this checking if the last byte of address is between 1 and 9?  Shouldn't it be the entire address between 1 and 9?

Something like:
```rust
address.0[0..19] == [0u8; 19] && (1..=9).contains(address.0[19])
```

Reply:
Fixed in [`b6d6434`](https://github.com/appliedzkp/zkevm-circuits/pull/312/commits/b6d6434dd5de09b31945e089102a875b0bafa10a).